wls-wahlvorbereitung-service/src/test/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/rest/unterbrechungsuhrzeit/UnterbrechungsUhrzeitControllerIntegrationTest.java

```diff
@@ -0,0 +1,212 @@
+            val request = get("/businessActions/unterbrechungsUhrzeit/" + wahlbezirkIDToFind);
+
+            val response = mockMvc.perform(request).andExpect(status().isOk()).andReturn();
+            val responseBodyAsDTO = objectMapper.readValue(response.getResponse().getContentAsString(), UnterbrechungsUhrzeitDTO.class);
+
+            Assertions.assertThat(responseBodyAsDTO).isEqualTo(expectedResponseBody);
+        }
+
+        @Test
+        @WithMockUser(
+                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT,
+                        Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT }
+        )
+        void noDataFound() throws Exception {
+
+            val wahlbezirkIDEmpty = "123";
+
+            val wahlbezirkIDNotEmpty = "456";
+            val unterbrechungsUhrzeitToFind = new UnterbrechungsUhrzeit();
+            unterbrechungsUhrzeitToFind.setWahlbezirkID(wahlbezirkIDNotEmpty);
+            unterbrechungsUhrzeitToFind.setUnterbrechungsUhrzeit(LocalDateTime.now().truncatedTo(ChronoUnit.MILLIS));
+            unterbrechungsUhrzeitRepository.save(unterbrechungsUhrzeitToFind);
+
+            val request = get("/businessActions/unterbrechungsUhrzeit/" + wahlbezirkIDEmpty);
+
+            val response = mockMvc.perform(request).andExpect(status().isNoContent()).andReturn();
+
+            Assertions.assertThat(response.getResponse().getContentAsString()).isEmpty();
+        }
+    }
+
+    @Nested
+    class PostUnterbrechungsUhrzeit {
+        @Test
+        @WithMockUser(
+                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT,
+                        Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT }
+        )
+        void newDataIsSaved() throws Exception {
+            val wahlbezirkID = "wahlbezirkID";
+            val writeDto = new UnterbrechungsUhrzeitWriteDTO(LocalDateTime.now().truncatedTo(ChronoUnit.MILLIS));
+            val request = buildPostRequest(wahlbezirkID, writeDto);
+
+            mockMvc.perform(request).andExpect(status().isCreated());
+
+            SecurityUtils.runAs(Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT);
+            val unterbrechungsUhrzeitFromRepo = unterbrechungsUhrzeitRepository.findById(wahlbezirkID).get();
+            val expectedUnterbrechungsUhrzeit = unterbrechungsUhrzeitModelMapper.toEntity(unterbrechungsUhrzeitDTOMapper.toModel(wahlbezirkID, writeDto));
+
+            Assertions.assertThat(unterbrechungsUhrzeitFromRepo).isEqualTo(expectedUnterbrechungsUhrzeit);
+        }
+
+        @Test
+        @WithMockUser(
+                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT,
+                        Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT }
+        )
+        void existingDataIsOverwritten() throws Exception {
+
+            val wahlbezirkID = "wahlbezirkID";
```

Comment:
aus meiner Sicht eine unnötige Leerzeile
```suggestion
        void existingDataIsOverwritten() throws Exception {
            val wahlbezirkID = "wahlbezirkID";
```